code/controllers/subsystem/ticker.dm

```diff
@@ -16,6 +16,10 @@ SUBSYSTEM_DEF(ticker)
 	var/start_immediately = FALSE
 	/// Boolean to track and check if our subsystem setup is done.
 	var/setup_done = FALSE
+	/// Boolean to delay game setup until at least one player is ready.
+	var/wait_for_players = TRUE
```

Comment:
Don't bother duplicating the variable, use `CONFIG_SET` instead

Reply:
YEah